src/asana/helpers.py

```diff
@@ -180,7 +180,13 @@ def _asana_user_url_from_github_user_handle(github_handle: str) -> Optional[str]
     user_id = _asana_user_id_from_github_handle(github_handle)
```

Comment:
might be helpful to add description for this function like there is `_asana_display_name_for_github_user`

Reply:
oh this is actually getting the asana `gid` from the dynamodb table not the display name which itself is a wrapper around `dynamodb_client.get_asana_domain_user_id_from_github_handle(github_handle)` but I'll remove the wrapper actually since it doesn't serve a purpose